src/properties/property.cpp

```diff
@@ -215,61 +256,48 @@ void Property::resetToUnchanged() {
     _isValueDirty = false;
 }
 
-std::string Property::generateJsonDescription() const {
-    const std::string cName = escapedJson(std::string(className()));
-    const std::string identifier = uri();
-    const std::string identifierSan = escapedJson(identifier);
-    const std::string& gName = guiName();
-    const std::string gNameSan = escapedJson(gName);
-    const std::string metaData = generateMetaDataJsonDescription();
-    const std::string description = generateAdditionalJsonDescription();
-
-    return std::format(
-        R"(
-{{"Type":"{}","Identifier":"{}","Name":"{}","MetaData":{},"AdditionalData":{}}}
-        )", cName, identifierSan, gNameSan, metaData, description
-    );
-}
-
-std::string Property::generateMetaDataJsonDescription() const {
+nlohmann::json Property::generateJsonDescription() const {
     static const std::unordered_map<Visibility, std::string> VisibilityConverter = {
-        { Visibility::Always, "Always" },
-        { Visibility::NoviceUser, "NoviceUser" },
-        { Visibility::User, "User" },
-        { Visibility::AdvancedUser, "AdvancedUser" },
-        { Visibility::Developer, "Developer" },
-        { Visibility::Hidden, "Hidden" }
+       { Visibility::Always, "Always" },
+       { Visibility::NoviceUser, "NoviceUser" },
+       { Visibility::User, "User" },
+       { Visibility::AdvancedUser, "AdvancedUser" },
+       { Visibility::Developer, "Developer" },
+       { Visibility::Hidden, "Hidden" }
```

Comment:
```suggestion
        { Visibility::Always, "Always" },
        { Visibility::NoviceUser, "NoviceUser" },
        { Visibility::User, "User" },
        { Visibility::AdvancedUser, "AdvancedUser" },
        { Visibility::Developer, "Developer" },
        { Visibility::Hidden, "Hidden" }
```

I think the indentation was correct before

Reply:
It's two tabs in now 🤔